lib/core/defaultPageCompleteCheck.js

```diff
@@ -2,9 +2,9 @@
 
 module.exports = `
 return (function(waitTime) {
-    try { 
-        const end = window.performance.timing.loadEventEnd;
-        return (end > 0) && (Date.now() > end + waitTime);
+    try {
```

Comment:
So we skip the extra wait time now, is there a reason for that? That is simple wait for the user to add more wait instead of changing the more complicated pageCompleteCheck